src/background/stores/path-snippet-store.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { StoreNames } from '../../common/stores/store-names';
+import { PathSnippetData } from '../../common/types/store-data/path-snippet-data';
+import { PathSnippetActions } from '../actions/path-snippet-actions';
+import { BaseStoreImpl } from './base-store-impl';
+
+export class PathSnippetStore extends BaseStoreImpl<PathSnippetData> {
+    private pathSnippetActions: PathSnippetActions;
```

Comment:
You can declutter this class a little bit by using [parameter properties](https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties). It would look like this
```typescript
...
// no need to declare the property here if using parameter properties on the constructor
// private pathSnippetActions: PathSnippetActions;

 constructor(private readonly pathSnippetActions: PathSnippetActions) {
        super(StoreNames.PathSnippetStore);

         // no need for this assignment anymore neither
         // this.pathSnippetActions = pathSnippetActions;
    }
```

Not all our classes follow this pattern because it was not introduce to our code base from the beginning but it's now the standard strategy to declare class properties.
I'm also adding the `readonly` modifier as the actions should not be change by this class.

Reply:
Great thank you!! Will implement!